### PR TITLE
Jupyter Notebook generation

### PR DESCRIPTION
**Pull request by Simcornelis**

### CODE REVIEW

File: npf/Graph.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+from npf.types.dataset import convert_to_xyeb
```
Comment (tbarbette) [inline npf/Graph.py:1]: In python filenames are lowercase.

File: npf/types/notebook/template.ipynb [hunk 2]
```diff
@@ -0,0 +1,322 @@
+{
```
Comment (tbarbette) [inline npf/types/notebook/template.ipynb:1]: Can this template be changed by the user easily? The path to this should be an overwriteable default.

**Review by tbarbette — CHANGES_REQUESTED** [review]

File: npf/grapher.py [hunk 3]
```diff
@@ -1030,6 +953,9 @@ def lam(x):
 
 
         versions = []
+        """Vars_all is the set of all variable combination that have some value. Taking the iperf case, it will be
```
Comment (tbarbette) [inline npf/grapher.py:956]: Thanks for those comments :)

File: npf/Graph.py (hunk 1 above)
Comment (Simcornelis) [reply npf/Graph.py:1]: Git didn't pick up on the Graph.py to graph.py renaming, so I'll force it.

File: npf/types/notebook/template.ipynb (hunk 2 above)
Comment (Simcornelis) [reply npf/types/notebook/template.ipynb:1]: Ok, working on it.

**Review by tbarbette — APPROVED** [review]

**Comment by tbarbette** [comment]
Semms ok! Thanks!

**Pull request merged**